py3-ntia-conformance-checker.yaml

```diff
@@ -2,7 +2,7 @@
 package:
   name: py3-ntia-conformance-checker
   version: 2.0.0
-  epoch: 0
+  epoch: 1
```

Comment:
Bumping this and not the other one because the current SBOM isn't actually valid until it's rebuilt.